src/ddscxx/include/dds/sub/LoanedSamples.hpp

```diff
@@ -121,6 +121,14 @@ class dds::sub::LoanedSamples
      */
     LoanedSamples(const LoanedSamples& other);
 
+    /**
+     * Copy-assigns a LoanedSamples instance to another.
+     *
+     * No actual data samples are copied.<br>
+     * Just references and reference counts are updated.
+     */
+    LoanedSamples& operator=(const LoanedSamples& other) = default;
+
```

Comment:
This is an official OMG spec file. I would be careful in modifying anything in here unless it is really critical, in which case we need to file an issue with the OMG. This does not seem to be one of those critical issues.

Reply:
After consultation with @reicheratwork I found out that this was needed in order to avoid compilation warnings due to the rule of three and rule of five. I think we will need to indeed file this an a spec issue with the OMG.